### PR TITLE
Story/classify hfi multiple days

### PR DESCRIPTION
**Pull request by conbrad**
- Replace `shapely` with `gdal.Warp` for reprojection
- Ran `api/advisory/db/database/tileserver_advisory_setup.sql` to add advisory table to tileserver
- Hook up frontend date picker state to vector tile query params
- Add mouse pointer hover event to show HFI polygon value

### CODE REVIEW

File: web/src/features/fba/components/fbaMap.test.tsx [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+import { DateTime } from 'luxon'
 import React from 'react'
 import { Provider } from 'react-redux'
 
 describe('FBAMap', () => {
   it('should render height with height and width properties set', () => {
     const { getByTestId } = render(
       <Provider store={store}>
-        <FBAMap selectedFireCenter={undefined} className={''} />
+        <FBAMap date={DateTime.fromISO('2016-05-25T09:08:34.123')} selectedFireCenter={undefined} className={''} />
```
Comment (Sybrand) [inline web/src/features/fba/components/fbaMap.test.tsx:12]: Can we use just the date part, instead if a time?
```suggestion
        <FBAMap date={DateTime.fromISO('2016-05-25')} selectedFireCenter={undefined} className={''} />
```

File: web/src/features/fba/pages/FireBehaviourAdvisoryPage.tsx [hunk 2]
```diff
@@ -24,8 +24,9 @@ const useStyles = makeStyles(() => ({
     height: 700
   },
   mapContainer: {
-    width: 900,
-    height: 700
+    width: '100%',
```
Comment (Sybrand) [inline web/src/features/fba/pages/FireBehaviourAdvisoryPage.tsx:27]: :)

**Review by Sybrand — CHANGES_REQUESTED** [review]
A few minor things - and could you address the code smells please? Nothing major structural.

**Review by Sybrand — APPROVED** [review]

File: api/advisory/db/database/tileserver.py [hunk 3]
```diff
@@ -8,36 +8,49 @@
 from app import config
 
 tileserver_read_user = config.get('TILESERVER_READ_USER', 'tileserv')
+tileserver_write_user = config.get('TILESERVER_WRITE_USER', 'tileserv')
 tileserver_postgres_password = config.get('TILESERVER_POSTGRES_PASSWORD', 'tileserv')
```
[the remark this answers is not in the document]
Comment (conbrad) [reply api/advisory/db/database/tileserver.py:12]: Not until post prototype

File: api/advisory/db/database/tileserver.py [hunk 4]
```diff
@@ -8,36 +8,49 @@
...
     max_overflow=int(config.get('POSTGRES_MAX_OVERFLOW', 10)),
     pool_pre_ping=True, connect_args={
         'options': '-c timezone=utc'})
 
+# connect to database - defaulting to always use utc timezone
+tile_server_write_engine = create_engine(TILESERVER_WRITE_STRING, connect_args={'options': '-c timezone=utc'})
 
-tile_server_session = sessionmaker(
-    autocommit=False, autoflush=False, bind=tile_server_engine)
+tile_server_read_session = sessionmaker(
```
[the remark this answers is not in the document]
Comment (conbrad) [reply api/advisory/db/database/tileserver.py:31]: We can address it before we merge to main, would rather not get hung up on this

**Review by andrea-williams — APPROVED** [review]

**Pull request merged**